chapter6/src/main/java/com/bsg5/chapter6/ArtistNotFoundException.java

```diff
@@ -0,0 +1,9 @@
+package com.bsg5.chapter6;
+
+public class ArtistNotFoundException extends RuntimeException {
+    private static long serialVersionUID = 1462190646166272903L;
```

Comment:
Our static analysis engine detects that your pull request includes a change of defining a `serialVersionUID` field that is not static. According to the [Java doc](https://docs.oracle.com/javase/8/docs/api/java/io/Serializable.html), the field should be made static if it is intended to specify the version UID for purposes of serialization.